airflow/models/dag.py

```diff
@@ -1469,7 +1515,7 @@ def create_dagrun(self,
                       run_id=None,
                       start_date=None,
                       external_trigger=False,
-                      conf=None,
+                      conf_dict=None,
```

Comment:
This is a breaking change, many users use keyword args

Reply:
True. Will go back with that - besides seems some cyclic imports need to be solve as well. I ill let the other "pylint" changes to be merged and will take a stab (I think one last time :) ) into this one ... We are basically 3 files to go :)

Reply:
Resolved.